From 48451db44516da1cbebd0690352fc298f8945676 Mon Sep 17 00:00:00 2001
From: Jiang Lu <lu.jiang@windriver.com>
Date: Wed, 19 Jun 2019 16:15:15 +0800
Subject: [PATCH 2/2] ARM:mach-imx:brace imx6 declaration with SOC_IMX6

Since imx6 specifc code if braced with SOC_IMX6, need brace variable declaratio to
prevnt compile warnings:

arch/arm/mach-imx/busfreq-imx.c: In function 'imx6ull_lower_cpu_rate':
arch/arm/mach-imx/busfreq-imx.c:202:6: warning: unused variable 'ret' [-Wunused-variable]
  int ret;
      ^~~
At top level:
arch/arm/mach-imx/busfreq-imx.c:70:29: warning: 'origin_soc_volt' defined but not used [-Wunused-variable]
 static int origin_arm_volt, origin_soc_volt;
                             ^~~~~~~~~~~~~~~
arch/arm/mach-imx/busfreq-imx.c:70:12: warning: 'origin_arm_volt' defined but not used [-Wunused-variable]
 static int origin_arm_volt, origin_soc_volt;
            ^~~~~~~~~~~~~~~

Signed-off-by: Jiang Lu <lu.jiang@windriver.com>
---
 arch/arm/mach-imx/busfreq-imx.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/mach-imx/busfreq-imx.c b/arch/arm/mach-imx/busfreq-imx.c
index da6220a1b884..e699a995a2c4 100644
--- a/arch/arm/mach-imx/busfreq-imx.c
+++ b/arch/arm/mach-imx/busfreq-imx.c
@@ -67,7 +67,9 @@ static int high_bus_count, med_bus_count, audio_bus_count, low_bus_count;
 static unsigned int ddr_low_rate;
 static int cur_bus_freq_mode;
 static u32 org_arm_rate;
+#ifdef CONFIG_SOC_IMX6
 static int origin_arm_volt, origin_soc_volt;
+#endif
 
 extern unsigned long iram_tlb_phys_addr;
 extern int unsigned long iram_tlb_base_addr;
@@ -199,7 +201,9 @@ static struct clk *origin_step_parent;
  */
 static void imx6ull_lower_cpu_rate(bool enter)
 {
+#ifdef CONFIG_SOC_IMX6
 	int ret;
+#endif
 
 	if (enter) {
 		org_arm_rate = clk_get_rate(arm_clk);
-- 
2.17.1

